boards/openmote-b/board.c

```diff
@@ -36,14 +36,14 @@ void board_init(void)
     gpio_init(RF24_SWITCH_CC2538_PIN, GPIO_OUT);
     gpio_init(RF24_SWITCH_AT86RF215_PIN, GPIO_OUT);
 
-#ifdef MODULE_AT86RF215
-    /* use at86rf215 2.4ghz radio*/
-    RF24_SWITCH_CC2538_OFF;
-    RF24_SWITCH_AT86RF215_ON;
-#else
+#ifdef MODULE_CC2538_RF
     /* use cc2538 2.4ghz radio*/
     RF24_SWITCH_CC2538_ON;
     RF24_SWITCH_AT86RF215_OFF;
```

Comment:
Maybe its better to switch OFF before switching on? Might not be good to have them both connected at the same time (een if short)

Reply:
Shouldn't make a difference since the radios aren't active yet, but I've changed it for consistency.